homeassistant/components/tedee/coordinator.py

```diff
@@ -103,6 +107,27 @@ async def _async_update(self, update_fn: Callable[[], Awaitable[None]]) -> None:
         except (TedeeClientException, TimeoutError) as ex:
             raise UpdateFailed("Querying API failed. Error: %s" % str(ex)) from ex
 
+    def webhook_received(self, message: dict[str, Any]) -> None:
+        """Handle webhook message."""
+        self.tedee_client.parse_webhook_message(message)
+        self.async_set_updated_data(self.tedee_client.locks_dict)
+
+    async def async_register_webhook(self, webhook_url: str) -> None:
+        """Register the webhook at the Tedee bridge."""
+        self.tedee_webhook_id = await self.tedee_client.register_webhook(webhook_url)
+
+    async def async_unregister_webhook(self) -> None:
+        """Unregister the webhook at the Tedee bridge."""
+        if self.tedee_webhook_id is not None:
+            try:
+                await self.tedee_client.delete_webhook(self.tedee_webhook_id)
+            except TedeeWebhookException as ex:
+                _LOGGER.exception(
+                    "Failed to unregister Tedee webhook from bridge: %s", ex
```

Comment:
```suggestion
                    "Failed to unregister Tedee webhook from bridge"
```
`_LOGGER.exception` already logs the stacktrace